integration-test/pom.xml

```diff
@@ -68,6 +68,12 @@
       <scope>test</scope>
       <classifier>classes</classifier>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.pnc</groupId>
+      <artifactId>rest-new</artifactId>
+      <scope>test</scope>
+      <classifier>classes</classifier>
+    </dependency>
```

Comment:
why is this needed? What does it do?

Reply:
It bundles all the rest-new classes, so that my test classes can have their @Injected objects working. Otherwise I suffered from NoClassFoundException IIRC